queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN 2016 AND 2019;
+SELECT name from animals WHERE neutered=True AND escape_attempts > 3;
```

Comment:
- [ ] This query is supposed to get the list of the name of all animals that are neutered and have `less than 3` escape attempts not `more than 3` so please fix that.

Reply:
Thanks or the review @Omar-Muhamad  changes made 👍